browser/js/common/directives/navbar/navbar.js

```diff
@@ -1,47 +1,71 @@
-app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state) {
+app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state, $window) {
 
-    return {
-        restrict: 'E',
-        scope: {},
-        templateUrl: 'js/common/directives/navbar/navbar.html',
+	return {
+		restrict: 'E',
+		scope: {},
+		templateUrl: 'js/common/directives/navbar/navbar.html',
 
-        link: function (scope) {
+		link: function (scope) {
+			var temp = 0;
+			scope.goToSection = function(val, state) {
+				if (val !== 'null') {
```

Comment:
why would it ever be equal to the string `null`? i think it's more likely that it will come in as the primitive value `undefined` (and maybe, in a very rare case -- though i can't figure one out -- come through as the primitive value of `null`), but i can't imagine that it would come through as a string.